packages/core/bootstrap/src/lib/modules/requester.ts

```diff
@@ -227,23 +273,39 @@ export class Requester {
     return data[path[0]]
   }
 
+  static addAdditionalFields(
```

Comment:
One thing we may want to be careful of is unintentionally overwriting the initialValue.

A quick check to see if `key` is already set, and if so give out a debug log, should make it easier to debug later if this comes up.